service/src/common/email-template-payload/base.email.payload.ts

```diff
@@ -1,6 +1,7 @@
 export interface BaseEmailPayload {
   platform: {
     url: string; // the url of alkemio
+    invitationsURL?: string; // the invitations url
```

Comment:
move this field to the specific email payload(s) - this is a very basic, generic payload, which everything inherits from.